DQM/TrackingMonitor/src/TrackAnalyzer.cc

```diff
@@ -259,8 +269,7 @@ void TrackAnalyzer::bookHistosForEfficiencyFromHitPatter(DQMStore::IBooker& iboo
     }
   }
 
-  edm::ESHandle<TrackerGeometry> trackerGeometry;
-  iSetup.get<TrackerDigiGeometryRecord>().get(trackerGeometry);
+  TrackerGeometry const& trackerGeometry = iSetup.get<TrackerDigiGeometryRecord>().get(trackerGeometryToken_);
```

Comment:
This could be just 
```suggestion
  TrackerGeometry const& trackerGeometry = iSetup.getData(trackerGeometryToken_);
```
no?

Reply:
Thanks much. That is better. I had forgotten about the new function getData. Saves some typing when ESGetTokens are used. No reason to type the record type again. I fixed this here and in the other places this occurs in this PR.

Reply:
Thanks!